kahuna/public/js/crop/controller.js

```diff
@@ -66,7 +67,16 @@ crop.controller('ImageCropCtrl',
         // else undefined is fine
     };
 
-    ctrl.crop = function() {
+     ctrl.callCrop = function() {
+         //prevents return keypress on the crop button posting crop twice
+         if (ctrl.cropping === false) {
+             crop();
+         } else {
+             return;
+         }
+     };
+
+    crop = () => {
```

Comment:
You need to define it as a `function` or else put a `const` before it, otherwise it's on the global scope. (Does our linting not catch that?)